DESCRIPTION

```diff
@@ -36,5 +36,7 @@ Suggests:
     lintr,
     rcmdcheck,
     rmarkdown,
+    spelling,
```

Comment:
if this is only used on CI, why add it to DESCRIPTION of the package? 

not the end of the world, but feels like a soft case of [test logic in production code smell](http://xunitpatterns.com/Test%20Logic%20in%20Production.html)

Reply:
It is added to "Suggests", a place that includes basically all dependencies that are not relevant to the package itself, but to the development process. So when a user needs to intentionally install `shiny.i18n` with suggested packages (and this basically means that this is done by someone who wants to develop the package).
Having packages in Suggests is convenient - it adds all required dependencies when the package is checked. It is also required by e.g. `pkgdown`: not in this particular case, but a line `spelling::spell_check_package(use_wordlist = TRUE)` added in the README would be enough to break documentation page build, if `spelling` was not included in the DESCRIPTION.